src/test/java/com/endava/internship/mocking/service/BasicValidationServiceTest.java

```diff
@@ -0,0 +1,45 @@
+package com.endava.internship.mocking.service;
+
+import com.endava.internship.mocking.model.Status;
+import com.endava.internship.mocking.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+
+class BasicValidationServiceTest {
+
+    private BasicValidationService service;
+
+    @BeforeEach
+    void setup(){
+         service = new BasicValidationService();
```

Comment:
Is it necessary to initialize the service before each test?

Reply:
removed